docs/mocksupport.rst

```diff
@@ -137,12 +137,13 @@ Pywbem mock does NOT support:
 1. CIM-XML protocol security and security constructor parameters of
    :class:`~pywbem.WBEMConnection`.
 2. Dynamic WBEM server providers in that the data for responses is from the
```

Comment:
Change "WBEM server providers" to "providers" and make it a link to the glossary term.

Reply:
Actually that whole paragraph was wrong in that we do support dynamic providers. Moved it to the supports section